www-client/kristall/kristall-0.7.3~git.recipe

```diff
@@ -6,16 +6,16 @@ Gopher, HTTP/HTTPS, and Finger protocols."
 HOMEPAGE="https://kristall.random-projects.net/"
 COPYRIGHT="2020 Felix Queißner"
 LICENSE="GNU GPL v3"
-REVISION="4"
-srcGitRev="a6aecaaa63037023e027f1273f99375f5067638c"
+REVISION="5"
+srcGitRev="bca592adf686b11321b7185f234525674d7d474c"
 SOURCE_URI="https://github.com/MasterQ32/kristall/archive/$srcGitRev.tar.gz"
-CHECKSUM_SHA256="4f7e3e4d6086a2f6a9e95f8052d3add833046ffb28a2e5371d713a454c1aa134"
+CHECKSUM_SHA256="326d3f3785ba5faf3caf12df1caf99e801816658b79881106e6b39204dfddfc5"
 SOURCE_FILENAME="kristall-$portVersion-tar.gz"
 SOURCE_DIR="kristall-$srcGitRev"
 ADDITIONAL_FILES="kristall.rdef.in"
 
-ARCHITECTURES="all !x86_gcc2"
-SECONDARY_ARCHITECTURES="?x86"
+ARCHITECTURES="!x86_gcc2 x86_64"
```

Comment:
@waddlesplash we'll be seeing this in the near future ...

Reply:
Indeed, something to watch out for.

Reply:
We can't all check architectures our selves (don't have a riscV/arm* environment for those), and for those some of these will depend on people with access to them to see if they actually build?

Reply:
The point of `all` is that most software will likely build on riscv or arm if it builds on x86_64. We don't necessarily need to check every single package on every architecture before we publish it; we can just see if it builds and fix it if it needs to be.